src/main/java/seedu/address/ui/SessionCard.java

```diff
@@ -37,12 +40,18 @@
     public SessionCard(Session session, int displayedIndex) {
         super(FXML);
         this.session = session;
+
         date.setText("Date: " + session.getSessionDate().getDate().toString());
         time.setText("Time: " + session.getSessionDate().getTime().toString());
         duration.setText("Duration: " + session.getDuration().getValue() + " mins");
         subject.setText("Subject: " + session.getSubject().getValue());
         fee.setText("Fee: $" + String.format("%.2f", session.getFee().getFee()));
         id.setText(displayedIndex + "");
+        if (session instanceof RecurringSession) {
+            sessionType.setText("R");
+        } else {
+            sessionType.setText("I");
```

Comment:
This is more of aesthetics, but would be great if we can do sth like:
`sessionType.setStyle("-fx-background-color: <color>");`
and give different colors for R & I.

Then, add padding for `sessionType` label in the fxml.

Just some aesthetic improvements, not necessary if too rush!